kolibri/utils/server.py

```diff
@@ -61,6 +61,12 @@ def start(port=8080):
     with open(PID_FILE, 'w') as f:
         f.write("%d\n%d" % (os.getpid(), port))
 
+    # TODO: not sure where it should ideally be located. Question is if it
+    # should be run every time the server is started or if it depends on some
+    # kind of state change.
```

Comment:
It gets triggered when state changes via the app itself, but state could also be changed by copying a `content` folder into `.kolibri`, or deleting a channel DB on disk, so it's best to have an extra call on start. It should be a pretty fast thing.